data_structures/Queue_implementation.cpp

```diff
@@ -0,0 +1,54 @@
+// CPP program to implement Queue using
+// two stacks with costly deQueue()
+#include <bits/stdc++.h>
+using namespace std;
+
+struct Queue {
+	stack<int> s1, s2;
```

Comment:
Please add a one-line description of what the variable is for (**see the example below**).

```c++
int64_t data = 0;       ///< The value/key of the node.
```

Reply:
ok Thank you for you suggestion , will correct it soon